mountpoint-s3/src/inode.rs

```diff
@@ -354,15 +352,16 @@ impl Superblock {
         }
 
         let expiry = Instant::now(); // TODO local inode stats never expire?
-                                     // Objects don't have an ETag until they are uploaded to S3
-        let stat = InodeStat::for_file(0, OffsetDateTime::now_utc(), expiry, None);
-        let kind = InodeKind::File;
+        let datetime = OffsetDateTime::now_utc();
+        let stat = match kind {
+            InodeKind::File => InodeStat::for_file(0, datetime, expiry, None), // Objects don't have an ETag until they are uploaded to S3
+            InodeKind::Directory => InodeStat::for_directory(datetime, expiry),
```

Comment:
lookup and readdir use mount time for directory stat, should we use the same value here?